mssqlcli/mssqlclioptionsparser.py

```diff
@@ -120,7 +120,8 @@ def create_parser():
         dest=u'multi_subnet_failover',
         action=u'store_true',
         default=False,
-        help=u'If application is connecting to AlwaysOn AG on different subnets, setting this provides faster detection and connection to currently active server.')
+        help=u'If application is connecting to AlwaysOn AG on different subnets, setting this provides faster '
```

Comment:
Let's revert this - it's already been changed and no point in changing it again.

Reply:
This is the one you asked me to revert back in previous PR, and I merged the PR before reverting back. So I reverted in this PR.

Reply:
This is your comment in the previous PR.
![image](https://user-images.githubusercontent.com/19577035/64292493-61027e80-cf1f-11e9-9f2c-96d5a2e7dd3d.png)